src/commands/sourcemaps/__tests__/upload.test.ts

```diff
@@ -81,6 +82,46 @@ describe('upload', () => {
       expect(write.mock.calls[0][0]).toContain('DATADOG_API_KEY')
     })
   })
+
+  describe('addRepositoryDataToPayloads', () => {
+    test('repository url and commit still defined without payload', async () => {
+      const command = new UploadCommand()
+      const write = jest.fn()
+      command.context = {stdout: {write}} as any
+      const payloads = new Array<Payload>({
+        cliVersion: '0.0.1',
+        minifiedFilePath: 'src/commands/sourcemaps/__tests__/fixtures-empty/empty.min.js',
+        minifiedUrl: 'http://example/empty.min.js',
+        projectPath: '',
+        service: 'svc',
+        sourcemapPath: 'src/commands/sourcemaps/__tests__/fixtures-empty/empty.min.js.map',
+        version: '1.2.3',
+      })
+      await command['addRepositoryDataToPayloads'](payloads)
+      expect(payloads[0].gitRepositoryURL).toBeDefined()
+      expect(payloads[0].gitCommitSha).toHaveLength(40)
+      expect(payloads[0].gitRepositoryPayload).toBeUndefined()
```

Comment:
Reading this test I don't understand exactly what's going on (not super familiar with this test codebase, that might also be a reason).

Could you add some doc to describe why we expect different results in both cases?
What do we expect to be in `getRepositoryURL`, `gitCommitSha`? Is it data from the current repository?

Reply:
Yes exactly. They are tests that run against the current repository with two kind of sourcemaps: an empty one and one which matches a tracked filename. The repositoryURL / commit is whatever the values are for the current repository when tests run.
I've added some comments to make it clearer.